corehq/apps/app_manager/templates/app_manager/partials/usercase_config.html

```diff
@@ -0,0 +1,151 @@
+{% load i18n %}
+
+<script type="text/html" id="usercase-config:case-properties:question">
+    <input class="input-large" type="hidden" data-bind="
+        questionsSelect: $root.getQuestions('all', false, true, true),
+        value: path,
+        optionsCaption: ' '
+    "/>
+    <!-- TODO: usercase_transaction not in scope: $root.getQuestions('all', false, usercase_transaction.allow.repeats(), true) -->
+    <p class="help-inline" data-bind="visible: required() && !path()">{% trans "Required" %}</p>
+    <p class="help-inline" data-bind="visible: !required() && !path() && key()">{% trans "Not assigned" %}</p>
+</script>
+
+<script type="text/html" id="usercase-config:case-properties:property">
```

Comment:
A lot of these look like identical copies for the ones in case_config_shared.html. Would be nice if we didn't have to copy them here.